Gulpfile.js

```diff
@@ -1,83 +1,78 @@
 const gulp = require('gulp');
 const $ = require('gulp-load-plugins')({
-  scope: ['devDependencies'],
-  lazy: true
+    scope: ['devDependencies'],
+    lazy: true
 });
-const connect = require('connect');
 const browserSync = require('browser-sync');
-const sStatic = require('serve-static');
+const process = require('process');
+
+var conf = require('./src/middleware/conf')(process.env.KOA_ENV);
 
 const JS_FILES = ['src/**/*.js'];
 const CSS_FILES = ['src/**/*.scss'];
 const HTML_FILES = ['src/**/*.html'];
 const TARGET = 'dist';
-const HTTP_PORT = 3000;
 
 // Vendor scripts, the order matters.
 const VENDOR_FILES = [
-  //'es6-shim/es6-shim.js',
-  'systemjs/dist/system-polyfills.js',
-  'angular2/bundles/angular2-polyfills.js',
-  'systemjs/dist/system.src.js',
-  'rxjs/bundles/Rx.js',
-  'angular2/bundles/angular2.dev.js',
-  'angular2/bundles/http.dev.js',
-  'angular2/bundles/router.dev.js'
+    //'es6-shim/es6-shim.js',
+    'systemjs/dist/system-polyfills.js',
+    'angular2/bundles/angular2-polyfills.js',
+    'systemjs/dist/system.src.js',
+    'rxjs/bundles/Rx.js',
+    'angular2/bundles/angular2.dev.js',
+    'angular2/bundles/http.dev.js'
 ].map((file) => 'node_modules/' + file);
 
 gulp.task('app-html', () => {
-  return gulp.src(HTML_FILES)
-    .pipe(gulp.dest(TARGET));
+    return gulp.src(HTML_FILES)
+        .pipe(gulp.dest(TARGET));
 });
 
 gulp.task('vendor-js', () => {
-  return gulp.src(VENDOR_FILES)
-    //.pipe($.uglify())
-    .pipe($.concatUtil('vendor.js'))
-    .pipe(gulp.dest(TARGET));
+    return gulp.src(VENDOR_FILES)
+        .pipe($.uglify())
+        .pipe($.concatUtil('vendor.js'))
+        .pipe(gulp.dest(TARGET));
 });
 
 gulp.task('app-js', () => {
-  return gulp.src(JS_FILES)
-    .pipe($.babel())
-    .pipe(gulp.dest(TARGET));
+    return gulp.src(JS_FILES)
+        .pipe($.babel())
+        .pipe(gulp.dest(TARGET));
 });
 
 gulp.task('app-css', () => {
-  return gulp.src(CSS_FILES)
-    .pipe($.sass().on('error', $.sass.logError))
-    .pipe($.autoprefixer())
-    //.pipe($.concatUtil('app.css'))
-    .pipe(gulp.dest(TARGET));
+    return gulp.src(CSS_FILES)
+        .pipe($.sass().on('error', $.sass.logError))
+        .pipe($.autoprefixer())
+        .pipe($.concatUtil('app.css'))
```

Comment:
@leudanielm No need to concat and get all styles in the same file. It looks more natural to have them in styles folder and separate files, as angular 2 have dynamic styles loader (maybe will help)